hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/ContainerProtocolCalls.java

```diff
@@ -84,7 +86,8 @@ private ContainerProtocolCalls() {
    * @throws IOException if there is an I/O error while performing the call
    */
   public static GetBlockResponseProto getBlock(XceiverClientSpi xceiverClient,
-      DatanodeBlockID datanodeBlockID) throws IOException {
+      DatanodeBlockID datanodeBlockID,
+      Collection<Token<? extends TokenIdentifier>> tokens) throws IOException {
```

Comment:
Can we update the javadoc comment of this method as we new added param tokens here?
The same suggestion for methods we changed below:

* putBlockAsync
* readChunk
* writeChunkAsync

Reply:
Thanks @linyiqun for the suggestion.  Would

```java
@param tokens list of tokens the current user has, possibly including a token for this block
```

be OK, or do you have a better description for this param?

Reply:
@adoroszlai , above description looks good to me. Thanks.

Reply:
We can directly encode the pass in token for the specific block id without additional token selection.